src/api/spec/jobs/send_event_emails_spec.rb

```diff
@@ -52,5 +52,9 @@
     it 'only creates two notifications' do
       expect(Notification.count).to eq(2)
     end
+
+    it 'creates a project log entry for the event' do
```

Comment:
You can also can check if the event was marked as logged.

If you implement what I also commented, then it should be in the model specs